Aqovia.PactProducerVerifier/Aqovia.PactProducerVerifier.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Aqovia.PactProducerVerifier</id>
+    <version>1.0.3</version>
```

Comment:
Shouldn't this just specify the major and minor version with the patch version left at 0 and set by Appveyor?